plugins/ecs-serviceconnect/config/netconfig.go

```diff
@@ -0,0 +1,199 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	utils "github.com/aws/amazon-vpc-cni-plugins/plugins/utils/config"
+	cniSkel "github.com/containernetworking/cni/pkg/skel"
+	cniTypes "github.com/containernetworking/cni/pkg/types"
+	"github.com/coreos/go-iptables/iptables"
+	"strconv"
```

Comment:
When importing packages, we follow the general golang convention of importing standards packages first ("fmt", "strings", ...), followed by packages internal to this repo ("github.com/aws/amazon-vpc-cni-plugins/..."), followed by 3rd party packages ("github.com/containernetworking/..."). See other plugins for an example.

Reply:
Thanks for the input. This is my first Go code. :) Modified it.